pkg/apis/events/v1alpha1/cloudbuildsource_conversion.go

```diff
@@ -31,8 +31,8 @@ func (source *CloudBuildSource) ConvertTo(_ context.Context, to apis.Convertible
 	switch sink := to.(type) {
 	case *v1beta1.CloudBuildSource:
 		sink.ObjectMeta = source.ObjectMeta
-		sink.Spec.PubSubSpec = convert.ToV1beta1PubSubSpec(source.Spec.PubSubSpec)
-		sink.Status.PubSubStatus = convert.ToV1beta1PubSubStatus(source.Status.PubSubStatus)
+		sink.Spec.PubSubSpec = convert.FromV1alpha1ToV1PubSubSpec(source.Spec.PubSubSpec)
```

Comment:
can we add a comment saying that v1beta1 uses v1 ducks or something like that... so that others don't get confused